src/Controller/OverviewController.php

```diff
@@ -182,6 +182,66 @@ public function add_category(Request $request): Response
     ]);
   }
 
+  /**
+   * The page for listing all categories in the system, wether they are active or not,
+   * and then providing simple means of toggling active/inactive.
+   * Will only accessible by admins
+   * 
+   * @author Daniel Boling
+   * 
+   * @Route("/category/list", name="list_category")
+   * @IsGranted("ROLE_USER")
+   */
+  public function list_category(Request $request): Response
+  {
+
+    $em = $this->getDoctrine()->getManager();
+
+    $categories = $this->getDoctrine()
+    // inits the database and Category table;
+    ->getRepository(Category::class)
+    ->findAll();
+
+    
+
+    return $this->render('list-category.html.twig', [
+      'categories' => $categories,
+      'date' => $this->date,
+    ]);
+  }
+  
+  /**
+   * Is called on button-click from twig file, updates active categories, and redirects to list_category
+   * 
+   * @author Daniel Boling
+   * @return redirect to list_category
+   * 
+   * @Route("/category/list/{id}", name="update_category")
+   */
+  public function categoryAction(Request $request, $id): Response
```

Comment:
This function name doesn't really describe what it does and isn't formatted properly according to https://github.com/goshencollege/gc-communicator/blob/main/STYLE_GUIDE.md#1-function-name.  Could you give it a better name?

Reply:
I figured this function name was required for something to work right I will change it, test for breaking changes, then update.

Reply:
Renamed to update_category to match route name.